website/snippets/_cloud-cli-relative-path.md

```diff
@@ -0,0 +1,20 @@
+The dbt Cloud CLI doesn't currently support relative paths in the [`packages.yml` file](/docs/build/packages).
+
+For example, trying to install [a local package](/docs/build/packages#local-packages) (`shared_macros` directory) in your `packages.yml` file won't work:
+
+```yaml
+# repository_root/my_dbt_project_in_a_subdirectory/packages.yml
+
+packages:
+  - local: ../shared_macros
+```
+
+In this example, `../shared_macros` is a relative path that tells dbt to look for:
+- `..` &mdash; Go one directory up (to repository_root).
+- `/shared_macros` &mdash; Find the `shared_macros` folder in the root directory.
+
+To work around this limitation:
+- Use the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud), which supports relative paths for this use case.
+- If using a monorepo with multiple projects as subdirectories, use [private packages](/docs/build/packages#private-packages) or the [git clone method](/docs/build/packages#ssh-key-method-command-line-only) to share package across projects.
```

Comment:
Did you test if private packages works with dbt Cloud CLI? Not sure at the top of my head :)

Reply:
i didn't but i was having a chat with Lucas Silbernagel about this and that's here i got it from. should i juts remove that bullet and just let users know to use the ide??